gatsby-node.js

```diff
@@ -207,94 +312,66 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
     })
   })
 
-  // Create contentVersion v2.0 pages
-  const contentV2Pages = [`eth`, `dapps`, `wallets/index`, `what-is-ethereum`]
-  const contentV2Languages = supportedLanguages.filter(
-    (lang) => getLangContentVersion(lang) >= 2.0
-  )
-  contentV2Pages.forEach((page) => {
-    const component = page
-    // Account for nested pages
-    if (page.includes("/index")) {
-      page = page.replace("/index", "")
-    }
-    contentV2Languages.forEach((lang) => {
-      createPage({
-        path: `/${lang}/${page}/`,
-        component: path.resolve(`./src/pages-conditional/${component}.js`),
-        context: {
-          slug: `/${lang}/${page}/`,
-          intl: {
-            language: lang,
-            languages: supportedLanguages,
-            defaultLanguage,
-            messages: getMessages("./src/intl/", lang),
-            routed: true,
-            originalPath: `/${page}/`,
-            redirect: false,
-          },
-        },
-      })
-    })
-  })
-
-  // Create contentVersion v1.0 pages
-  // v1.0 doesn't have existing markdown files for these pages
-  const contentV1Pages = [`eth`, `dapps`, `wallets/index`]
-  const contentV1Languages = supportedLanguages.filter(
-    (lang) => getLangContentVersion(lang) === 1.0
-  )
-  contentV1Pages.forEach((page) => {
-    const component = page
-    // Account for nested pages
-    if (page.includes("/index")) {
-      page = page.replace("/index", "")
-    }
-    contentV1Languages.forEach((lang) => {
-      createPage({
-        path: `/${lang}/${page}/`,
-        component: path.resolve(`./src/pages-conditional/${component}.js`),
-        context: {
-          slug: `/${lang}/${page}/`,
-          isContentEnglish: true,
-          intl: {
-            language: lang,
-            languages: supportedLanguages,
-            defaultLanguage,
-            messages: getMessages("./src/intl/", lang),
-            routed: true,
-            originalPath: `/${page}/`,
-            redirect: false,
+  // Create `/pages-conditional/` pages for each language unless a markdown page already exists.
+  // This avoids overwriting markdown pages with a component page of the same name.
+  // Note: once all these markdown pages have been replaced with updated JSON translation files,
+  // we can remove this logic and the `/pages-conditional/` directory.
+  const outdatedMarkdown = [`eth`, `dapps`, `wallets`, `what-is-ethereum`]
+  outdatedMarkdown.forEach((page) => {
+    supportedLanguages.forEach((lang) => {
+      const markdownPath = `${__dirname}/src/content/translations/${lang}/${page}/index.md`
+      const langHasOutdatedMarkdown = fs.existsSync(
+        markdownPath,
+        fs.constants.R_OK | fs.constants.W_OK
```

Comment:
Is this 2nd param needed? What's it do? I notice we don't use it in `fs.existsSync` elsewhere.

Reply:
Probably not. But using existsSync to check if the file exists is all.